widgetssdk/src/androidTest/java/com/glia/widgets/DownloadsFolderDataSourceTest.java

```diff
@@ -0,0 +1,59 @@
+package com.glia.widgets;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.platform.app.InstrumentationRegistry;
+
+import com.glia.widgets.filepreview.data.source.local.DownloadsFolderDataSource;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.ByteArrayInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+
+@RunWith(AndroidJUnit4.class)
+public class DownloadsFolderDataSourceTest {
```

Comment:
Why there is only a single test case for every method. Is it impossible to cover this class with more tests?